mkdocs/contrib/search/prebuild-index.js

```diff
@@ -0,0 +1,53 @@
+var lunr = require('./templates/search/lunr'),
+    stdin = process.stdin,
+    stdout = process.stdout,
+    buffer = [];
+
+stdin.resume();
+stdin.setEncoding('utf8');
+
+stdin.on('data', function (data) {
+  buffer.push(data);
+});
+
+stdin.on('end', function () {
+  var data = JSON.parse(buffer.join('')),
+      lang = ['en'];
+
+  if (data.config) {
+    if (data.config.lang && data.config.lang.length) {
+      lang = data.config.lang;
+      if (lang.length > 1 || lang[0] !== "en") {
+        require('./lunr-language/lunr.stemmer.support')(lunr);
+        if (lang.length > 1) {
+          require('./lunr-language/lunr.multi')(lunr);
+        }
+        for (var i=0; i < lang.length; i++) {
+          if (lang[i] != 'en') {
+            require('./lunr-language/lunr.' + lang[i])(lunr);
+          }
+        }
+      }
+    }
+    if (data.config.separator && data.config.separator.length) {
+      lunr.tokenizer.separator = new RegExp(data.config.separator);
+    }
+  }
+
+  var idx = lunr(function () {
+    if (lang.length === 1 && lang[0] !== "en" && lunr[lang[0]]) {
+      this.use(lunr[lang[0]]);
+    } else if (lang.length > 1) {
+      this.use(lunr.multiLanguage.apply(null, lang));
+    }
+    this.field('title', { boost: 10 });
```

Comment:
As has been mentioned in https://github.com/mkdocs/mkdocs/issues/1319 and https://github.com/olivernn/lunr.js/issues/312 field boosts are not currently supported by Lunr 2.x. The second parameter here will be ignored, I'm not sure that does any harm, but perhaps a comment might help.

When I get around to implementing field boosts it will almost certainly use this interface (same as Lunr 1.x) so you _could_ leave this here to take advantage of field boosts when they arrive.

Reply:
Thanks. I realize boost is being ignored. But your right, no point in leaving it there. It will just cause confusion later when we forget that.